tutorials/cpi-sentiment-analysis-s4/cpi-sentiment-analysis-s4.md

```diff
@@ -64,31 +64,31 @@ In the User for Inbound Communication section add the technical user which was c
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 3: ](Create marketing master data integration)]  
+[ACCORDION-BEGIN [Step 3: ](Creating Marketing Master Data integration)]  
 
 For communication with the OData Web service a technical user must be created in the SAP Hybris Marketing Cloud System.
 This user needs to have two communication scenarios assigned:  
 
  - SAP_COM_0003 Marketing – Master Data Integration   
  - SAP_COM_0004 Marketing – Business Data Integration  
 
-Log on to your SAP Hybris Marketing Cloud system.  
+Logon to your SAP Hybris Marketing Cloud System.  
 ![s/4 logon](1.png)
 
-In the SAP Fiori launchpad, select the **Communication Arrangements** tile.  
+In the SAP Fiori launchpad select the **Communication Arrangements** tile.  
 ![communication Arrangements](9.png)  
 
 Choose **New** to create a new communication arrangement.
 ![communication Arrangements popup](10.png)  
 
-Select **SAP_COM_0003 (Marketing - Master Data Integration)** and choose **Create**.
+Select **(SAP_COM_0003 (Marketing - Master Data Integration)** and choose **Create**.
 ![mdi](11.png)  
 
-Under **Common Data**, select the Communication System via value help which was created in step 2.
+Under Common Data, select the Communication System via value help which was created in step 2.
 
 ![system](12.png)  
 
-Under **Inbound Communication**, the technical user that was created in step 1 needs to be entered. This field should be automatically filled in.  
+Under Inbound Communication the technical user which was created in step 1. This field should be automatically filled in.  
```

Comment:
"Under Inbound Communication the technical user which was created"... this is confusing.  I think either the subject and object are reversed, or there is a section of the sentence missing.  Should this be:
_The technical user, which was created in step 1, should be automatically filled in under Inbound Communication_